cmd/remote-storage/app/server.go

```diff
@@ -81,10 +80,6 @@ func createGRPCHandler(f storageFactory, logger *zap.Logger) (*shared.GRPCHandle
 		StreamingSpanWriter: func() spanstore.Writer { return nil },
 	}
 
-	ar, aw := f.InitArchiveStorage(logger)
-	impl.ArchiveSpanReader = func() spanstore.Reader { return ar }
-	impl.ArchiveSpanWriter = func() spanstore.Writer { return aw }
```

Comment:
@yurishkuro Does this change make sense? My thinking was that now two servers need to be started if the grpc storage itself needs to be configured as archive. But what if someone wants the archive functionality from es storage when starting their grpc remote storage server?

Reply:
> archive functionality from es storage when starting their grpc remote storage server

do you mean running `remote-storage` with `storage_type=es`? We said they need to run two `remote-storage` instances, and the "archive" behavior in ES can now be reproduced via input parameters, can it not? Are you asking about v1 or v2?

Reply:
> do you mean running remote-storage with storage_type=es

Yeah that's what I was asking about 

> We said they need to run two remote-storage instances, and the "archive" behavior in ES can now be reproduced via input parameters, can it not? 

Yeah it can - I was mostly asking about v1. This works as intended for v2 but for v1 it would mean the users need to change their input parameters, right?

Reply:
Yes, since they need to customize the "archive" instance. Maybe worth mentioning this in the breaking changes section of the previous PR.